### DISCUSSION TITLE
v2.11.6

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#219] [General]
Fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.11.6'>v2.11.6</a>.</em>

**Comment by inspron** [comment]
First, thank you for developing the VPN Hotspot app. It is incredibly helpful. I use it daily.

I have one feature that I would kindly ask you to consider:

Could we have an option to *always* be in "monitored" mode? 

The app has root permission, but with every restart I have to go back into the app to re-enable the "monitored" mode. Occasionally, the app would lose the monitor mode and would have be manually re-enabled. 

It would be amazing if the app could always be in monitored mode. 

Thank you for your time and consideration.

**Reply by Mygod** [reply]
Hi this is on my list but it is not very high priority. Feel free to contribute to this repo since it is open source.
